prow/Makefile

```diff
@@ -157,9 +148,3 @@ marque-service:
 	kubectl apply -f cluster/marque_service.yaml
 
 .PHONY: hook-image hook-deployment hook-service test-pr-image sinker-image sinker-deployment deck-image deck-deployment deck-service splice-image splice-deployment marque-image marque-deployment marque-service
-
-update-godeps:
```

Comment:
(We should probably add an update-godeps script somewhere, but this code is just dead)